back/engines/commercial/multi_tenancy/app/services/multi_tenancy/tenant_template_service.rb

```diff
@@ -15,7 +15,7 @@ class TenantTemplateService
       Comment
     ].to_set.freeze
 
-    SKIP_IMAGE_PRESENCE_VALIDATION = %w[IdeaImage].freeze
+    SKIP_IMAGE_PRESENCE_VALIDATION = %w[IdeaImage ContentBuilder::LayoutImage].freeze
```

Comment:
We want to skip the new validation of presence of :image for `layout_images` here (as otherwise the validation will cause the template application to fail. For more details see https://github.com/CitizenLabDotCo/citizenlab/pull/3902)